packages/manager/apps/telecom/src/app/app.js

```diff
@@ -40,9 +41,14 @@ import carrierSip from './telecom/telephony/carrierSip';
 import errorPage from './error-page/error-page.module';
 import navbar from '../components/navbar';
 
+import 'ovh-ui-kit/dist/oui.css';
```

Comment:
Sounds strange that this isn't already done

Reply:
https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/app.less#L4